08-projects.Rmd

```diff
@@ -4615,11 +4615,11 @@ dbGetQuery(con, "SELECT * FROM battings LIMIT 5;")
 - The result of running the R code.
 ```
 
-##### 3. How many times has Giancarlo Stanton struck out in years in which he played for "MIA" or "FLO"?
+##### 3. How many times in total has Giancarlo Stanton struck out in years in which he played for "MIA" or "FLO"?
 
 **Hint:** Use the `people` table to find the `playerID` and use the `batting` table to find the statistics.
 
-**Relevant topics:** dbGetQuery, AND/OR, COUNT, SUM
+**Relevant topics:** dbGetQuery, AND/OR, SUM
```

Comment:
While [the solution](https://github.com/TheDataMine/fall2020-stat29000/blob/master/project10/stat29000project10solutions.Rmd) includes one line of code that uses `COUNT`, it seems that it is not necessary.